### DISCUSSION TITLE
Enable multiple call-to-action buttons in nav bar.

### DISCUSSION THREAD

**Discussion by leewynne** [forem/forem#16211] [Feature Requests]
There is an existing Forem that has slightly different functionality when a member hits 'create' post. The options are, create post, create listing.

This could be further extended to create post, share media (youtube link), create listing.

[Here is a post by Ben that explains this in more detail ](https://forem.dev/ben/how-to-properly-enable-multiple-call-to-action-buttons-in-nav-bar-4dba)

**Comment by Maurice8426** [comment]
Yeah, you are right. As few days ago. I posted there about [best youtube automation tools](https://www.ihorhnatewicz.com/best-youtube-automation-tools/) because I was facing some sort of issue with it. As there were not such option of share media etc so due to which no one provided us the solution about our issue. And still the same the we are stuck with that issue.
